### PR TITLE
Remove letouzey from CODEOWNERS since he left the Coq organization.

### PR DESCRIPTION
**Pull request by SkySkimmer**
This PR assumes that secondary maintainers are willing to become primary, concerned are @maximedenes @herbelin @ppedrot @mattam82 @gares @ejgallego 
Let's give them time to react if that's not accurate.

It also leaves omega/romega without any maintainers.

We should find secondary maintainers for those components but IMO there's no point delaying this PR until it can be done.

### CODE REVIEW

**Review by ejgallego — APPROVED** [review]
I am OK.

**Review by ppedrot — APPROVED** [review]

**Review by maximedenes — CHANGES_REQUESTED** [review]
I can't be the primary maintainer for extraction. There are many bugs to address and I don't have enough time at the moment. There should either be a real primary maintainer, or we should declare it unmaintained.

**Comment by Zimmi48** [comment]
Primary maintainers are about merging PRs, not fixing bugs though.

**Comment by maximedenes** [comment]
> Primary maintainers are about merging PRs, not fixing bugs though.

It may be the current status quo (although I don't think that's really the case), but I think we should aim for a workflow where maintainers maintain (and also merge PRs).

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -159,10 +158,6 @@
 /plugins/nsatz/         @thery
 # Secondary maintainer @ppedrot
 
-/plugins/omega/         @letouzey
-
-/plugins/romega/        @letouzey
```
Comment (ejgallego) [inline .github/CODEOWNERS]: Can't you keep this and use `@nobody`?

Comment (Zimmi48) [reply .github/CODEOWNERS]: `@nobody` is a real person though so you probably meant `@ghost`.

**Comment by Zimmi48** [comment]
> It may be the current status quo (although I don't think that's really the case), but I think we should aim for a workflow where maintainers maintain (and also merge PRs).

That's not the deal that was made when people agreed to be code owner. So we can change our terminology to distinguish code owner and maintainer if you want, but I don't think we should aim for such a workflow, at least not in the medium-term.

**Comment by Zimmi48** [comment]
So you may refuse to become the primary code owner, but you can't use this argument as it sends a bad signal to other people already or about to become code owners.

File: .github/CODEOWNERS (hunk 1 above)
Comment (gares) [reply .github/CODEOWNERS]: The other day I discovered you can cite "teams", eg `@coq/pushers`, so maybe this is what we should do...

**Comment by gares** [comment]
> we can change our terminology to distinguish code owner and maintainer if you want

The confusion pissed me off already quite a few times, and Maxime is aware of that.

For example I'm "maintaining" CoqIDE that I think should have never existed in the first place (why on earth would one write a UI in OCaml?). I can surely review and merge patches on CoqIDE, since I know its code, but expecting me to fix bugs that are not super critical is stretching my involvement too much. 

I think we should at least clarify it at the next WG. Having two roles may confuse users, that refer to maintainers as "people that may know", according to my experience, not people that are expected to do.

**Comment by SkySkimmer** [comment]
>I can't be the primary maintainer for extraction. There are many bugs to address and I don't have enough time at the moment. There should either be a real primary maintainer, or we should declare it unmaintained.

Removed.

I also added a default entry `/* @coq/pushers`, no clue if it will work.

**Comment by Zimmi48** [comment]
> I also added a default entry /* @coq/pushers, no clue if it will work.

I would do some experimentation elsewhere before doing so. I'm afraid that it means it is going to ask a review from everyone in the team everytime a file without code owner is modified (for instance every time `CHANGES` is modified).

**Comment by SkySkimmer** [comment]
Here's a manual review request for pushers, any effect on your side?

**Comment by Zimmi48** [comment]
Yes, similar to the one I get when I am personally requested a review.

**Review by gares — APPROVED** [review]

**Comment by maximedenes** [comment]
> The confusion pissed me off already quite a few times, and Maxime is aware of that.
> 
> For example I'm "maintaining" CoqIDE that I think should have never existed in the first place (why on earth would one write a UI in OCaml?). I can surely review and merge patches on CoqIDE, since I know its code, but expecting me to fix bugs that are not super critical is stretching my involvement too much.
> 
> I think we should at least clarify it at the next WG. Having two roles may confuse users, that refer to maintainers as "people that may know", according to my experience, not people that are expected to do.

Fair enough. So maybe we should also list maintainers. For example, assignments for issues should target maintainers, and PR assignments should be target code owners. Let's discuss it separately.

**Review by maximedenes — APPROVED** [review]

**Comment by maximedenes** [comment]
> maximedenes approved these changes on behalf of coq/pushers

Wait! Why does GitHub say that? I'm supposed to be an owner of this file.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1,6 +1,9 @@
 # This file describes the maintainers for the main components. See
 # `dev/doc/MERGING.md`.
 
+# default
+/*      @coq/pushers
```
Comment (SkySkimmer) [inline .github/CODEOWNERS]: Do keep this change?

**Comment by Zimmi48** [comment]
It's because @SkySkimmer also requested a review from the team. Therefore, it's not requesting a review from me (as a member of the team) anymore.

File: .github/CODEOWNERS (hunk 2 above)
Comment (Zimmi48) [reply .github/CODEOWNERS]: No!

Comment (maximedenes) [reply .github/CODEOWNERS]: Maybe not, it seems to have strange interactions with individual owners.

Comment (maximedenes) [reply .github/CODEOWNERS]: In fact, you can put me back for extraction, then we clarify the distinction between maintainers and owners. Sorry for the noise.

Comment (Zimmi48) [reply .github/CODEOWNERS]: My main concern is not to spam code owners who are not watching the repository and thus should only receive e-mails / notifications when their part of the code is affected.

**Review by maximedenes — APPROVED** [review]

**Pull request merged**